src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -461,7 +460,12 @@ public ListBoxModel doFillCommitInfoChoiceItems() {
 
         public ListBoxModel doFillTokenCredentialIdItems(@AncestorInPath Item context) {
 
-            if(context == null && !Jenkins.getInstance().hasPermission(Jenkins.ADMINISTER) ||
+            Jenkins j = Jenkins.getInstance();
```

Comment:
getActiveInstance maybe?
its deprecated in newer jenkins core, but the replacement isn't available to 2.98
https://javadoc.jenkins-ci.org/jenkins/model/Jenkins.html#getActiveInstance--

Reply:
+1

Reply:
Had another look, seems like on this version getInstance is the best choice, getActiveInstance is deprecated on this version too. 
Unfortunately it has a `@Nonnull` annotation on it but it can be null, as there's a system property required to be enabled to make it throw the required exception

Reply:
It is deprecated, but it won't be removed from API. Once you update the core dependency to 2.107+, you can start using `Jenkins.get()` instead